core/src/main/java/org/kiwix/kiwixmobile/core/history/adapter/HistoryListItem.kt

```diff
@@ -36,6 +37,7 @@ sealed class HistoryListItem {
     val historyTitle: String,
     val dateString: String,
     val timeStamp: Long,
+    var isSelected: Boolean,
```

Comment:
you could introduce a default value so you can ignore having to update the other constructors